hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -135,7 +135,7 @@ const COLUMNS = [
     isVisible: true,
     sorter: (a: IDatanode, b: IDatanode) => a.lastHeartbeat - b.lastHeartbeat,
     render: (heartbeat: number) => {
-      return heartbeat > 0 ? moment(heartbeat).format('lll') : 'NA';
+      return heartbeat > 0 ? moment(heartbeat).format('YYYY/MM/DD HH:mm:ss') : 'NA';
```

Comment:
Can we use `ll LTS`?
```suggestion
      return heartbeat > 0 ? moment(heartbeat).format('ll LTS') : 'NA';
```